parm/config/config.resources

```diff
@@ -737,7 +737,7 @@ elif [ ${step} = "esfc" ]; then
 elif [ ${step} = "efcs" ]; then
 
     if [ ${CASE} = "C768" ]; then
-      export wtime_efcs="01:20:00"
+      export wtime_efcs="06:00:00"
```

Comment:
In fact, using "01:20:00" is only good for coarse resolution, if we want to run C768/C384, then the walltime is too short. Therefore, I increase the wall time. I agree that we may have an another variable to cover early cycle. However, in the early-enkf, we may not run gfsemn jobs since it will be run GEFSv13.